.github/workflows/verible-ci.yml

```diff
@@ -43,23 +45,38 @@ jobs:
             FOLDER: /tmp/pages
             CLEAN: true
 
+
+   Matrix:
+      runs-on: ubuntu-latest
+      outputs:
+        matrix: ${{ steps.generate.outputs.matrix }}
+      steps:
+      - uses: actions/checkout@v2
+      - id: generate
+        run: |
+          matrix='['
+          for item in releasing/*/; do
+            matrix+='{"OS": "'"`echo $(basename $item) | cut -d- -f1`"'", "OS_VERSION": "'"`echo $(basename $item) | cut -d- -f2`"'"},'
+          done
+          matrix+=']'
+          echo "$matrix"
+          echo "::set-output name=matrix::$matrix"
+
+
    Release:
+      needs: Matrix
       runs-on: ubuntu-16.04
       strategy:
         fail-fast: false
         matrix:
-          env:
-            - { MODE: bin, OS: ubuntu, OS_VERSION: xenial }
-            - { MODE: bin, OS: ubuntu, OS_VERSION: bionic }
-            - { MODE: bin, OS: ubuntu, OS_VERSION: eoan }
-            - { MODE: bin, OS: ubuntu, OS_VERSION: focal }
-            - { MODE: bin, OS: ubuntu, OS_VERSION: groovy }
-#           - { MODE: bin, OS: centos, OS_VERSION: 6 }
```

Comment:
centos 6 was actually commented out in the previous GHA config - I guess we should delete its dir so its not included in the matrix generation.

Reply:
Done.